app/src/main/java/com/asfoundation/wallet/App.kt

```diff
@@ -114,6 +115,9 @@ class App : MultiDexApplication(), BillingDependenciesProvider {
   @Inject
   lateinit var ewtObtainer: EwtAuthenticatorService
 
+  @Inject
+  lateinit var oemIdExtractorService: OemIdExtractorService
```

Comment:
There is extra logic to obtain the oemid inside the PartnerAddressService. Might be better to use the PartnerAddressService.getAttributionEntity method instead of the OemIdExtractorService